src/Skoruba.IdentityServer4.Admin.BusinessLogic.Identity/Services/IdentityService.cs

```diff
@@ -435,19 +464,19 @@ public virtual async Task<TRoleClaimsDto> GetRoleClaimsAsync(string roleId, int
             await AuditEventLogger.LogEventAsync(new RoleClaimsRequestedEvent<TRoleClaimsDto>(roleClaimDtos));
 
             return roleClaimDtos;
-        }
-
-		public virtual async Task<TRoleClaimsDto> GetUserRoleClaimsAsync(string userId, string claimSearchText, int page = 1, int pageSize = 10)
-		{
-			var userExists = await IdentityRepository.ExistsUserAsync(userId);
-			if (!userExists) throw new UserFriendlyErrorPageException(string.Format(IdentityServiceResources.UserDoesNotExist().Description, userId), IdentityServiceResources.UserDoesNotExist().Description);
-
-			var identityRoleClaims = await IdentityRepository.GetUserRoleClaimsAsync(userId, claimSearchText, page, pageSize);
-			var roleClaimDtos = Mapper.Map<TRoleClaimsDto>(identityRoleClaims);
-
-			return roleClaimDtos;
-		}
-
+        }
+
+		public virtual async Task<TRoleClaimsDto> GetUserRoleClaimsAsync(string userId, string claimSearchText, int page = 1, int pageSize = 10)
```

Comment:
Whitespace only.. Seems the dev branch was indented a bit farther than surrounding code.